adcgen/expression/polynom_container.py

```diff
@@ -198,6 +198,40 @@ def rename_tensor(self, current: str, new: str,
             renamed = ExprContainer(inner=renamed, **self.assumptions)
         return renamed
 
+    def expand_coulomb_ri(self, factorisation: Literal['sym', 'asym'] = 'sym',
+                          wrap_result: bool = True) -> "Expr | ExprContainer":
+        """
+        Fatorises the symmetric ERIs in chemist notation into an RI format.
+        Note that this expands the polynomial to account for the uniqueness
+        of each RI auxilliary index.
+
+        Args:
+            factorisation : str, optional
+                Which type of factorisation to use (sym or asym).
+                Defaults to 'sym'
+            wrap_result : bool, optional
+                Whether to wrap the result in an ExprContainer.
+                Defaults to True.
+
+        Returns:
+            Expr | ExprContainer: The fatorised result
+        """
+        from .expr_container import ExprContainer
+
+        factorised = S.One
+        for _ in range(int(self.exponent)):
```

Comment:
Same here for the exponent. But I think here we dont need the loop. It should be fine to simply forward the call and init a `Pow` object with the result.

Reply:
But something like  B^P_{pq} B^P_{rs} )^2 = (pq|rs) (a single tensor with a exponent) would be handled by `ObjectContainer`.
Only [ (pq|rs) + (ab|cd) ]^2 would be handled by the `PolynomContainer`. And here I think the Pow object is actually what we want